getting-started/00.OVERVIEW.md

```diff
@@ -0,0 +1,19 @@
+- [**00.OVERVIEW**](./00.OVERVIEW.md)
```

Comment:
I personally feel like a title should always begin the document, seeing the table of contents first without any context feels odd to me.

Reply:
This also matches msdocs rules, to have H1 at the very top of the document.

Reply:
fixed